laundry-management-service/app/build.gradle

```diff
@@ -21,12 +21,14 @@ bootJar {
 sourceCompatibility = 1.8
 
 repositories {
+    mavenLocal()
```

Comment:
is it necessary?

Reply:
yes, otherwise gradle will search the library in central repositories